frontend/apps/filemanager/filemanager.lua

```diff
@@ -157,6 +157,30 @@ function FileManager:init()
                         UIManager:close(self.file_dialog)
                     end,
                 },
+                {
+                    text = _("Purge .sdr"),
+                    enabled = lfs.attributes(util.realpath(file), "mode") == "file" and util.realpath(file):match("%.") and
+                            lfs.attributes(util.realpath(file):match("(.*)%.")..".sdr", "mode") == "directory",
```

Comment:
Better use getSidecarDir to get the sidecard directory path. You can find examples within this file.